examples/angularjs-perf/index.html

```diff
@@ -8,45 +8,51 @@
 		<style>[ng-cloak] { display: none; }</style>
 	</head>
 	<body>
-		<section id="todoapp" ng-controller="TodoCtrl">
+		<section id="todoapp" ng-controller="TodoCtrl as TC">
```

Comment:
I should also mention the frequent use of `vm` as Controller's alias, which stands for ViewModel. The problem with it - it does not scale beyond single controllers:

```html
<div ng-controller="ParentCtrl as vm">
   <div ng-controller="ChildCtrl as vm">
    ... plenty of markup ...
        <p>I love {{vm.fruit}}</p>
    ... plenty of markup ...
   </div>
    ... plenty of markup ...
   <p>Actually I love {{vm.fruit}}</p>
    ... plenty of markup ...
 </div>
```
Do you still easily see which `vm` is where???

It was recommended by John Papa in 
https://github.com/johnpapa/angular-styleguide#controlleras-with-vm
but he was careful to point below (which many readers seem to skip):

> Note: When working with larger codebases, using a more descriptive name can help ease cognitive overhead & searchability. Avoid overly verbose names that are cumbersome to type.
```html
<!-- recommended -->
<input ng-model="productVm.id">
```

In the light of this, I could rename `TC` into `todoCtrlVm` or `tCtrl` or `tCVm`? 

Which one is better:
```
ng-blur="todoCtrl.doneEditing(todo, $index)" 
ng-blur="tCtrl.doneEditing(todo, $index)" 
ng-blur="tVm.doneEditing(todo, $index)" 
ng-blur="tC.doneEditing(todo, $index)" 
```